### PR TITLE
Tabnet bug fixed

### PR DESCRIPTION
**Pull request by javaThonc**
Fix Tabnet's bug that it will not be stopped at the right epoch and adding alpha360 example of tabnet

### CODE REVIEW

File: examples/highfreq/high_freq_tree.ipynb [hunk 1]
```diff
@@ -0,0 +1,192 @@
+{
```
Comment (you-n-g) [inline examples/highfreq/high_freq_tree.ipynb:1]: Why is this file necessary?

File: examples/highfreq/highfreq_gdbt_model.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
```
Comment (you-n-g) [inline examples/highfreq/highfreq_gdbt_model.py:2]: Why is this file necessary?

**Review by you-n-g — COMMENTED** [review]
@javaThonc 
Could you update Tabnet's results in the benchmark table?

**Comment by D-X-Y** [comment]
It seems that TabNet does not use the best-validation-performance model for evaluation?

File: examples/highfreq/high_freq_tree.ipynb (hunk 1 above)
Comment (javaThonc) [reply examples/highfreq/high_freq_tree.ipynb:1]: This file is a demo for high-frequency workflow to run in Jupiter notebook and I think it might be straightforward for others to have a python demo.

Comment (javaThonc) [reply examples/highfreq/high_freq_tree.ipynb:1]: Since it is really similar to the current workflow example and I will upload a YAML version of it later,   we can also delete it to make the example more concise, depending on whether it is necessary to have a high-frequency version of workflow.

**Comment by javaThonc** [comment]
Close the pr since there is overlapping between prs
